src/game/chopper.js

```diff
@@ -6,20 +6,23 @@ import Resources from './resources';
 export default ex.Actor.extend({
   constructor(engine) {
     const centerX = engine.drawWidth / 2;
+    this.kills = 0;
     // const centerY = engine.drawHeight / 2;
     ex.Actor.apply(this, [
       centerX,
       -150, // -centerY * Settings.scale.y,
-      Settings.CHOPPER_WIDTH,
-      Settings.CHOPPER_HEIGHT,
+      96 * Settings.scale.x,
```

Comment:
Is this the only place `CHOPPER_WIDTH`, `CHOPPER_HEIGHT` was used? If so, perhaps update those with the values you have here?

Reply:
Yes they were actually used here but didn't have a value in settings, moving it to settings now, much nicer.